src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -156,6 +159,76 @@ BinaryCompactObject::BinaryCompactObject(
         context,
         "Cannot have periodic boundary conditions with a binary domain");
   }
+
+  // Create block names and groups
+  static std::array<std::string, 6> wedges{"UpperZ", "LowerZ", "UpperY",
```

Comment:
I think `wedges` can be renamed to `wedge_directions` or `axis_directions` or similar